tomni/cytosmart_data_format/annotations/polygon/main.py

```diff
@@ -101,6 +109,36 @@ def perimeter(self) -> float:
     def points(self) -> List[Point]:
         return self._points
 
+    @points.setter
```

Comment:
There should not exist a setter for the `points`, imho. There purpose is a mostly dumb data model class. I'd rather do this in the init with a util function. Then you have also a unit testable function only for simplification.